worker/mozillaEvaluationWorker/mozillaEvaluationWorker.go

```diff
@@ -33,25 +34,32 @@ var sstlsURL = "https://statics.tls.security.mozilla.org/server-side-tls-conf.js
 var sstls ServerSideTLSJson
 var modern, intermediate, old Configuration
 
+var once sync.Once
+
 var log = logger.GetLogger()
 
 func init() {
+	worker.RegisterWorker(workerName, worker.Info{Runner: new(eval), Description: workerDesc})
+}
+
+func loadConfigurations() {
 	err := getConffromURL(sstlsURL)
 	if err != nil {
-		log.Error(err)
-		log.Error("Could not get tls confs from url - fallback to locally saved configurations")
+		log.Warn(err)
+		log.Warn("Could not get tls confs from url - fallback to locally saved configurations")
 		// Try to continue with the locally hosted TLS configurations
 		err = json.Unmarshal([]byte(ServerSideTLSConfiguration), &sstls)
 		if err != nil {
 			log.Error(err)
 			log.Error("Could not load Server Side TLS configuration. Evaluation Worker not available")
+			//if we reach this stage, no TLS configs are loaded, so there is no need to have this worker available.
+			worker.RemoveWorker(workerName)
```

Comment:
I don't think this is a valid scenario either, since fallback configurations are embedded in the code.
I will keep the `RemoveWorker` function in the worker just for consistency.